install.ps1

```diff
@@ -877,18 +667,181 @@ if (-not $noGui.IsPresent) {
         $newXmlNode.SetAttribute("name", "RAW_TOOLS_DIR")
         $newXmlNode.SetAttribute("value", $rawToolsDirText.text)
 
-        # Add selected packages
-        $packages = $configXml.SelectSingleNode('//packages')
-        foreach($package in $selectedPackagesBox.Items) {
-            $newXmlNode = $packages.AppendChild($configXml.CreateElement("package"))
-            $newXmlNode.SetAttribute("name", $package)
-        }
+
+        [void]$formEnv.Close()
+
     } else {
         Write-Host "[+] Cancel pressed, stopping installation..."
         Start-Sleep 3
         exit 1
     }
 
+    ################################################################################
+    ## PACKAGE SELECTION BY CATEGORY
+    ################################################################################
+
+    function Get-Packages-Categories {
```

Comment:
@sara-rn 
> @Ana06 there is no need to use `Find-Package`, therefore the installation of nuget or download `categories.txt` is not necessary. At the end of the feed from https://www.myget.org/F/vm-packages/Packages there is a `<link rel="next"` with the URL for the next packages.
The packages are selected and the Continue button adds the selected items to the `config.xml`

Seeing how complicated the code to use `https://www.myget.org/F/vm-packages/api/v2/Packages?$filter=IsLatestVersion%20eq%20true` is, I think the solution with `Find-Package` was better: much simpler code and easier to maintain.